experiment/compatibility-versions/e2e-k8s-compatibility-versions.sh

```diff
@@ -70,8 +70,8 @@ trap signal_handler INT TERM
 
 # build kubernetes / node image, e2e binaries
 build() {
-  # build the node image w/ kubernetes
-  kind build node-image -v 1
+  # build the node image w/ kubernetes  (NOT USED FOR INITIAL CLUSTER, but for upgrade)
+
   # Ginkgo v1 is used by Kubernetes 1.24 and earlier, fallback if v2 is not available.
```

Comment:
non-blocking: you could drop this, since compat version is never going to be used that far back

Reply:
Done, removed Ginkgo v1 logic (for k8s 1.24 and earlier which this test won't run against)